### ISSUE TITLE
Error while running `cljr-add-project-dependency`

### ISSUE THREAD

**Issue by BrunoBonacci** [clojure-emacs/clj-refactor.el#449]
## Expected behaviour

find and add library to classpath, relates to: #435 but incorrectly closed

## Actual behavior

Exception is raised:

```
cljr--get-error-value: Error in nrepl-refactor: clojure.lang.Compiler$CompilerException: java.lang.ClassNotFoundException: javax.xml.bind.DatatypeConverter, compiling:(mranderson048/http_kit/v2v2v0/org/httpkit/client.clj:1:1)
 at clojure.lang.Compiler.load (Compiler.java:7526)
    clojure.lang.RT.loadResourceScript (RT.java:379)
    clojure.lang.RT.loadResourceScript (RT.java:370)
    clojure.lang.RT.load (RT.java:460)
    clojure.lang.RT.load (RT.java:426)
    clojure.core$load$fn__6548.invoke (core.clj:6046)
    clojure.core$load.invokeStatic (core.clj:6045)
    clojure.core$load.doInvoke (core.clj:6029)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.core$load_one.invokeStatic (core.clj:5848)
    clojure.core$load_one.invoke (core.clj:5843)
    clojure.core$load_lib$fn__6493.invoke (core.clj:5888)
    clojure.core$load_lib.invokeStatic (core.clj:5887)
    clojure.core$load_lib.doInvoke (core.clj:5868)
    clojure.lang.RestFn.applyTo (RestFn.java:142)
    clojure.core$apply.invokeStatic (core.clj:659)
    clojure.core$load_libs.invokeStatic (core.clj:5925)
    clojure.core$load_libs.doInvoke (core.clj:5909)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:659)
    clojure.core$require.invokeStatic (core.clj:5947)
    clojure.core$require.doInvoke (core.clj:5947)
    clojure.lang.RestFn.invoke (RestFn.java:619)
    refactor_nrepl.artifacts$eval7133$loading__6434__auto____7134.invoke (artifacts.clj:1)
    refactor_nrepl.artifacts$eval7133.invokeStatic (artifacts.clj:1)
    refactor_nrepl.artifacts$eval7133.invoke (artifacts.clj:1)
    clojure.lang.Compiler.eval (Compiler.java:7062)
    clojure.lang.Compiler.eval (Compiler.java:7051)
    clojure.lang.Compiler.load (Compiler.java:7514)
    clojure.lang.RT.loadResourceScript (RT.java:379)
    clojure.lang.RT.loadResourceScript (RT.java:370)
    clojure.lang.RT.load (RT.java:460)
    clojure.lang.RT.load (RT.java:426)
    clojure.core$load$fn__6548.invoke (core.clj:6046)
    clojure.core$load.invokeStatic (core.clj:6045)
    clojure.core$load.doInvoke (core.clj:6029)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.core$load_one.invokeStatic (core.clj:5848)
    clojure.core$load_one.invoke (core.clj:5843)
    clojure.core$load_lib$fn__6493.invoke (core.clj:5888)
    clojure.core$load_lib.invokeStatic (core.clj:5887)
    clojure.core$load_lib.doInvoke (core.clj:5868)
    clojure.lang.RestFn.applyTo (RestFn.java:142)
    clojure.core$apply.invokeStatic (core.clj:659)
    clojure.core$load_libs.invokeStatic (core.clj:5925)
    clojure.core$load_libs.doInvoke (core.clj:5909)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:659)
    clojure.core$require.invokeStatic (core.clj:5947)
    clojure.core$require.doInvoke (core.clj:5947)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    refactor_nrepl.middleware$require_and_resolve.invokeStatic (middleware.clj:25)
    refactor_nrepl.middleware$require_and_resolve.invoke (middleware.clj:24)
    refactor_nrepl.middleware$fn__5917.invokeStatic (middleware.clj:108)
    refactor_nrepl.middleware/fn (middleware.clj:108)
    clojure.lang.Delay.deref (Delay.java:42)
    clojure.core$deref.invokeStatic (core.clj:2312)
    clojure.core$deref.invoke (core.clj:2298)
    refactor_nrepl.middleware$artifact_list_reply.invokeStatic (middleware.clj:117)
    refactor_nrepl.middleware$artifact_list_reply.invoke (middleware.clj:116)
    refactor_nrepl.middleware$wrap_refactor$fn__5980.invoke (middleware.clj:214)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_slurp$fn__6104.invoke (nrepl.clj:94)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_macroexpand$fn__6180.invoke (nrepl.clj:243)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_ns$fn__6188.invoke (nrepl.clj:256)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_apropos$fn__6112.invoke (nrepl.clj:102)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_info$fn__6162.invoke (nrepl.clj:180)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_spec$fn__6228.invoke (nrepl.clj:372)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.interruptible_eval$interruptible_eval$fn__944.invoke (interruptible_eval.clj:144)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_out$fn__6196.invoke (nrepl.clj:288)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.load_file$wrap_load_file$fn__982.invoke (load_file.clj:81)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_tracker$fn__6260.invoke (nrepl.clj:435)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_content_type$fn__6096.invoke (nrepl.clj:82)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_debug$fn__6138.invoke (nrepl.clj:136)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_enlighten$fn__6146.invoke (nrepl.clj:162)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_inspect$fn__6172.invoke (nrepl.clj:199)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.caught$wrap_caught$fn__890.invoke (caught.clj:97)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.session$add_stdin$fn__1093.invoke (session.clj:326)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    refactor_test.core$eval6353$fn__6354$fn__6356.invoke (form-init2565016382356426869.clj:1)
    cider.nrepl$wrap_profile$fn__6204.invoke (nrepl.clj:297)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_resource$fn__6220.invoke (nrepl.clj:360)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_test$fn__6244.invoke (nrepl.clj:398)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.session$session$fn__1078.invoke (session.clj:272)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.print$wrap_print$fn__857.invoke (print.clj:234)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.server$handle_STAR_.invokeStatic (server.clj:18)
    nrepl.server$handle_STAR_.invoke (server.clj:15)
    nrepl.server$handle$fn__1115.invoke (server.clj:27)
    clojure.core$binding_conveyor_fn$fn__5476.invoke (core.clj:2022)
    clojure.lang.AFn.call (AFn.java:18)
    java.util.concurrent.FutureTask.run (FutureTask.java:264)
    java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1128)
    java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:628)
    java.lang.Thread.run (Thread.java:835)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.DatatypeConverter
 at java.net.URLClassLoader.findClass (URLClassLoader.java:436)
    clojure.lang.DynamicClassLoader.findClass (DynamicClassLoader.java:69)
    java.lang.ClassLoader.loadClass (ClassLoader.java:588)
    clojure.lang.DynamicClassLoader.loadClass (DynamicClassLoader.java:77)
    java.lang.ClassLoader.loadClass (ClassLoader.java:521)
    java.lang.Class.forName0 (Class.java:-2)
    java.lang.Class.forName (Class.java:415)
    clojure.lang.RT.classForName (RT.java:2204)
    clojure.lang.RT.classForNameNonLoading (RT.java:2217)
    mranderson048.http_kit.v2v2v0.org.httpkit.client$eval9222$loading__6434__auto____9223.invoke (client.clj:1)
    mranderson048.http_kit.v2v2v0.org.httpkit.client$eval9222.invokeStatic (client.clj:1)
    mranderson048.http_kit.v2v2v0.org.httpkit.client$eval9222.invoke (client.clj:1)
    clojure.lang.Compiler.eval (Compiler.java:7062)
    clojure.lang.Compiler.eval (Compiler.java:7051)
    clojure.lang.Compiler.load (Compiler.java:7514)
    clojure.lang.RT.loadResourceScript (RT.java:379)
    clojure.lang.RT.loadResourceScript (RT.java:370)
    clojure.lang.RT.load (RT.java:460)
    clojure.lang.RT.load (RT.java:426)
    clojure.core$load$fn__6548.invoke (core.clj:6046)
    clojure.core$load.invokeStatic (core.clj:6045)
    clojure.core$load.doInvoke (core.clj:6029)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.core$load_one.invokeStatic (core.clj:5848)
    clojure.core$load_one.invoke (core.clj:5843)
    clojure.core$load_lib$fn__6493.invoke (core.clj:5888)
    clojure.core$load_lib.invokeStatic (core.clj:5887)
    clojure.core$load_lib.doInvoke (core.clj:5868)
    clojure.lang.RestFn.applyTo (RestFn.java:142)
    clojure.core$apply.invokeStatic (core.clj:659)
    clojure.core$load_libs.invokeStatic (core.clj:5925)
    clojure.core$load_libs.doInvoke (core.clj:5909)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:659)
    clojure.core$require.invokeStatic (core.clj:5947)
    clojure.core$require.doInvoke (core.clj:5947)
    clojure.lang.RestFn.invoke (RestFn.java:619)
    refactor_nrepl.artifacts$eval7133$loading__6434__auto____7134.invoke (artifacts.clj:1)
    refactor_nrepl.artifacts$eval7133.invokeStatic (artifacts.clj:1)
    refactor_nrepl.artifacts$eval7133.invoke (artifacts.clj:1)
    clojure.lang.Compiler.eval (Compiler.java:7062)
    clojure.lang.Compiler.eval (Compiler.java:7051)
    clojure.lang.Compiler.load (Compiler.java:7514)
    clojure.lang.RT.loadResourceScript (RT.java:379)
    clojure.lang.RT.loadResourceScript (RT.java:370)
    clojure.lang.RT.load (RT.java:460)
    clojure.lang.RT.load (RT.java:426)
    clojure.core$load$fn__6548.invoke (core.clj:6046)
    clojure.core$load.invokeStatic (core.clj:6045)
    clojure.core$load.doInvoke (core.clj:6029)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.core$load_one.invokeStatic (core.clj:5848)
    clojure.core$load_one.invoke (core.clj:5843)
    clojure.core$load_lib$fn__6493.invoke (core.clj:5888)
    clojure.core$load_lib.invokeStatic (core.clj:5887)
    clojure.core$load_lib.doInvoke (core.clj:5868)
    clojure.lang.RestFn.applyTo (RestFn.java:142)
    clojure.core$apply.invokeStatic (core.clj:659)
    clojure.core$load_libs.invokeStatic (core.clj:5925)
    clojure.core$load_libs.doInvoke (core.clj:5909)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:659)
    clojure.core$require.invokeStatic (core.clj:5947)
    clojure.core$require.doInvoke (core.clj:5947)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    refactor_nrepl.middleware$require_and_resolve.invokeStatic (middleware.clj:25)
    refactor_nrepl.middleware$require_and_resolve.invoke (middleware.clj:24)
    refactor_nrepl.middleware$fn__5917.invokeStatic (middleware.clj:108)
    refactor_nrepl.middleware/fn (middleware.clj:108)
    clojure.lang.Delay.deref (Delay.java:42)
    clojure.core$deref.invokeStatic (core.clj:2312)
    clojure.core$deref.invoke (core.clj:2298)
    refactor_nrepl.middleware$artifact_list_reply.invokeStatic (middleware.clj:117)
    refactor_nrepl.middleware$artifact_list_reply.invoke (middleware.clj:116)
    refactor_nrepl.middleware$wrap_refactor$fn__5980.invoke (middleware.clj:214)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_slurp$fn__6104.invoke (nrepl.clj:94)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_macroexpand$fn__6180.invoke (nrepl.clj:243)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_ns$fn__6188.invoke (nrepl.clj:256)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_apropos$fn__6112.invoke (nrepl.clj:102)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_info$fn__6162.invoke (nrepl.clj:180)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_spec$fn__6228.invoke (nrepl.clj:372)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.interruptible_eval$interruptible_eval$fn__944.invoke (interruptible_eval.clj:144)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_out$fn__6196.invoke (nrepl.clj:288)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.load_file$wrap_load_file$fn__982.invoke (load_file.clj:81)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_tracker$fn__6260.invoke (nrepl.clj:435)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_content_type$fn__6096.invoke (nrepl.clj:82)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_debug$fn__6138.invoke (nrepl.clj:136)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_enlighten$fn__6146.invoke (nrepl.clj:162)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_inspect$fn__6172.invoke (nrepl.clj:199)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.caught$wrap_caught$fn__890.invoke (caught.clj:97)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.session$add_stdin$fn__1093.invoke (session.clj:326)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    refactor_test.core$eval6353$fn__6354$fn__6356.invoke (form-init2565016382356426869.clj:1)
    cider.nrepl$wrap_profile$fn__6204.invoke (nrepl.clj:297)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_resource$fn__6220.invoke (nrepl.clj:360)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_test$fn__6244.invoke (nrepl.clj:398)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.session$session$fn__1078.invoke (session.clj:272)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.print$wrap_print$fn__857.invoke (print.clj:234)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.server$handle_STAR_.invokeStatic (server.clj:18)
    nrepl.server$handle_STAR_.invoke (server.clj:15)
    nrepl.server$handle$fn__1115.invoke (server.clj:27)
    clojure.core$binding_conveyor_fn$fn__5476.invoke (core.clj:2022)
    clojure.lang.AFn.call (AFn.java:18)
    java.util.concurrent.FutureTask.run (FutureTask.java:264)
    java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1128)
    java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:628)
    java.lang.Thread.run (Thread.java:835)
```

## Steps to reproduce the problem

  * lein new test-project
  * Open `core.clj` with Emacs
  * start CIDER `C-c M-j`
  * load namespace `C-c C-k`
  * attempt to add a library with `M-x cljr-add-project-dependency`

## Environment & Version information

```
;; CIDER 0.21.0 (New York), nREPL 0.6.0
;; Clojure 1.9.0, Java 12
clj-refactor 2.4.0
```

### Leiningen or Boot version

`Leiningen 2.8.3 on Java 12 OpenJDK 64-Bit Server VM`

### Emacs version

`Emacs 26.2`

### Operating system

`System Version: macOS 10.14.5`

Using JDK8 I get the following error

```
Retrieving list of available libraries...
cljr--get-error-value: Error in nrepl-refactor: java.util.concurrent.ExecutionException: java.lang.RuntimeException: Invalid token: trident/1.1
 at java.util.concurrent.FutureTask.report (FutureTask.java:122)
    java.util.concurrent.FutureTask.get (FutureTask.java:192)
    clojure.core$deref_future.invokeStatic (core.clj:2292)
    clojure.core$future_call$reify__8097.deref (core.clj:6894)
    clojure.core$deref.invokeStatic (core.clj:2312)
    clojure.core$deref.invoke (core.clj:2298)
    refactor_nrepl.artifacts$update_artifact_cache_BANG_.invokeStatic (artifacts.clj:91)
    refactor_nrepl.artifacts$update_artifact_cache_BANG_.invoke (artifacts.clj:87)
    refactor_nrepl.artifacts$artifact_list.invokeStatic (artifacts.clj:97)
    refactor_nrepl.artifacts$artifact_list.invoke (artifacts.clj:94)
    clojure.lang.Var.invoke (Var.java:381)
    refactor_nrepl.middleware$artifact_list_reply.invokeStatic (middleware.clj:117)
    refactor_nrepl.middleware$artifact_list_reply.invoke (middleware.clj:116)
    refactor_nrepl.middleware$wrap_refactor$fn__6108.invoke (middleware.clj:214)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_inspect$fn__6300.invoke (nrepl.clj:199)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_apropos$fn__6240.invoke (nrepl.clj:102)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_trace$fn__6380.invoke (nrepl.clj:417)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_test$fn__6372.invoke (nrepl.clj:398)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.caught$wrap_caught$fn__890.invoke (caught.clj:97)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_info$fn__6290.invoke (nrepl.clj:180)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_debug$fn__6266.invoke (nrepl.clj:136)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.middleware.print$wrap_print$fn__857.invoke (print.clj:234)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_enlighten$fn__6274.invoke (nrepl.clj:162)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_tracker$fn__6388.invoke (nrepl.clj:435)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_profile$fn__6332.invoke (nrepl.clj:297)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_version$fn__6404.invoke (nrepl.clj:454)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_slurp$fn__6232.invoke (nrepl.clj:94)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    cider.nrepl$wrap_spec$fn__6356.invoke (nrepl.clj:372)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    refactor_test.core$eval6481$fn__6482$fn__6484.invoke (form-init9582691157512346995.clj:1)
    nrepl.middleware.session$session$fn__1078.invoke (session.clj:272)
    nrepl.middleware$wrap_conj_descriptor$fn__633.invoke (middleware.clj:16)
    nrepl.server$handle_STAR_.invokeStatic (server.clj:18)
    nrepl.server$handle_STAR_.invoke (server.clj:15)
    nrepl.server$handle$fn__1115.invoke (server.clj:27)
    clojure.core$binding_conveyor_fn$fn__5476.invoke (core.clj:2022)
    clojure.lang.AFn.call (AFn.java:18)
    java.util.concurrent.FutureTask.run (FutureTask.java:266)
    java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1149)
    java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:624)
    java.lang.Thread.run (Thread.java:748)
Caused by: java.lang.RuntimeException: Invalid token: trident/1.1
 at clojure.lang.Util.runtimeException (Util.java:221)
    clojure.lang.EdnReader.interpretToken (EdnReader.java:285)
    clojure.lang.EdnReader.read (EdnReader.java:171)
    clojure.lang.EdnReader.readDelimitedList (EdnReader.java:766)
    clojure.lang.EdnReader$VectorReader.invoke (EdnReader.java:672)
    clojure.lang.EdnReader.read (EdnReader.java:145)
    clojure.lang.EdnReader.read (EdnReader.java:111)
    clojure.lang.EdnReader.readString (EdnReader.java:67)
    clojure.edn$read_string.invokeStatic (edn.clj:46)
    clojure.edn$read_string.invokeStatic (edn.clj:37)
    clojure.edn$read_string.invoke (edn.clj:37)
    clojure.core$map$fn__5587.invoke (core.clj:2747)
    clojure.lang.LazySeq.sval (LazySeq.java:40)
    clojure.lang.LazySeq.seq (LazySeq.java:49)
    clojure.lang.Cons.next (Cons.java:39)
    clojure.lang.RT.next (RT.java:706)
    clojure.core$next__5108.invokeStatic (core.clj:64)
    clojure.core.protocols$fn__7852.invokeStatic (protocols.clj:169)
    clojure.core.protocols/fn (protocols.clj:124)
    clojure.core.protocols$fn__7807$G__7802__7816.invoke (protocols.clj:19)
    clojure.core.protocols$seq_reduce.invokeStatic (protocols.clj:31)
    clojure.core.protocols$fn__7835.invokeStatic (protocols.clj:75)
    clojure.core.protocols/fn (protocols.clj:75)
    clojure.core.protocols$fn__7781$G__7776__7794.invoke (protocols.clj:13)
    clojure.core$reduce.invokeStatic (core.clj:6748)
    clojure.core$reduce.invoke (core.clj:6730)
    refactor_nrepl.artifacts$get_artifacts_from_clojars_BANG_.invokeStatic (artifacts.clj:83)
    refactor_nrepl.artifacts$get_artifacts_from_clojars_BANG_.invoke (artifacts.clj:81)
    refactor_nrepl.artifacts$update_artifact_cache_BANG_$fn__10975.invoke (artifacts.clj:89)
    clojure.core$binding_conveyor_fn$fn__5476.invoke (core.clj:2022)
    clojure.lang.AFn.call (AFn.java:18)
    java.util.concurrent.FutureTask.run (FutureTask.java:266)
    java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1149)
    java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:624)
    java.lang.Thread.run (Thread.java:748)
```

**Comment by expez** [comment]
>  relates to: #435 but incorrectly closed

The other issues says fixed in 2.4.1-SNAPSHOT, but you're running 2.4.0, right?

**Comment by BrunoBonacci** [comment]
yes, running the "stable" version: 2.4.0
